code/src/main/java/com/onlinecontacttracing/messaging/filters/LengthFlaggingFilter.java

```diff
@@ -0,0 +1,21 @@
+package com.onlinecontacttracing.messaging.filters;
+
+import com.onlinecontacttracing.messaging.filters.FlaggingFilter;
+import com.onlinecontacttracing.storage.PositiveUser;
+import com.onlinecontacttracing.storage.PotentialContact;
+import java.lang.Exception;
+
+//Checks that the length of the message is within limits
+public class LengthFlaggingFilter implements FlaggingFilter{
+  private static int limitNumOfCharacters = 500;
+  
+  public boolean passesFilter(PositiveUser positiveUser, String message) {
+    if (message.length() > limitNumOfCharacters) {
```

Comment:
Same as comment below. You can just return message.length() <= limitNumOfCharacters